main.tex

```diff
@@ -54,27 +60,32 @@
 
 \begin{enumerate}[label=(\arabic*)]
 
-\item  An Owner or Occupier of a Lot must not use the Lot, or permit it to be used, so as to cause a hazard to the health, safety and security of an Owner, Occupier, or user of another Lot.
-
-\item  An Owner or Occupier of a Lot must not use a Lot or the Common Property for any purpose which may:
+\item  An Owner or Occupier of a Lot must not use the Lot, or permit it to be used, so as to cause a hazard to the health, safety and security of an Owner, Occupier, pet, or human guest of another Lot.
+\newline
+\newline This rule does not apply to scenarios where the \textit{hazard} is:
 
 \begin{enumerate}[label=(\alph*)]
+\item  The growth of a plant
+\item  The expression of political opinion
+\end{enumerate}
+
+\item  An Owner or Occupier of a Lot must not use a Lot or the Common Property for any purpose which is:
 
-\item  be contrary to any Law;
+\begin{enumerate}[label=(\alph*)]
 
-\item  adversely affect the reputation of the Development;
+\item  contrary to any Law from \href{https://www.merri-bek.vic.gov.au/}{Merri-bek City Council}, the \href{https://www.vic.gov.au/}{Victorian Government}, or the \href{https://my.gov.au/en/services}{Australian federal government};
 
-\item  be reasonably likely to cause or contribute to any loss or damage of a Lot or the Common Property; and
+\item  reasonably likely to cause or contribute to any loss or damage of a Lot or the Common Property; and
 
-\item  be reasonably likely to cause or contribute to any nuisance or hazard.
+\item  reasonably likely to cause or contribute to any hazard.
 
 \end{enumerate}
 
-\item  An Owner or Occupier of a Lot must not permit any child under the age of 14 years, and owner whom the Owner or Occupier has control, to remain unaccompanied (by a responsible adult) on Common Property, including car parking areas and/or recreational areas (if applicable), where there is a hazard or possible danger.
```

Comment:
The parking is not just for cars.

Leaving children unaccompanied is covered in the later section about common property.